### PR TITLE
Add Agitoon

### LINKED ISSUE #5596: [Site Request] agit185
**Name of the website**
agit185

**Website urls (examples below)**
- Site: https://agit185.com/
- Manga List: https://agit185.com/azi_toon/11005.html
- Manga example: https://agit185.com/azi_toons/11005/1167925.html
- Chapter online viewer example: https://agit185.com/azi_toons/11005/1167925.html

**Languages**
List of languages supported by the website
Korea
**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### PR DESCRIPTION
**Pull request by MikeZeDev**
Fixes https://github.com/manga-download/hakuneko/issues/5596.

Website was fun to add. Method used is to circumvent redirections in iframes that mess up with fetchUI.

### CODE REVIEW

File: src/web/mjs/connectors/Agitoon.mjs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    async _initializeConnector() {
+        await this._getNeededValues();
+    }
+
+    async _getNeededValues() {
+        //https://agit185.com/data/azitoon.js?v=0.42317244219852657
+        //contains
+
+        /*
```
Comment (27136773) [inline src/web/mjs/connectors/Agitoon.mjs]: May remove comments without explanative value

File: src/web/mjs/connectors/Agitoon.mjs [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        var img_domain3 = "https://image.blacktoonimg.com/";
+        var img_domain4 = "https://image2.blackmanaimg.com/";
+        var img_domain5 = "https://image3.blackmanaimg.com/";
+        var img_per = "5";
+        var img_per2 = "20";
+        var img_per3 = "5";
+        var img_per4 = "35";
+        var img_per5 = "35";
+        */
+
```
Comment (27136773) [inline src/web/mjs/connectors/Agitoon.mjs]: suggestion to clear `this.cdns ` and `this.img_per` before filling with current values.

**Review by 27136773 — APPROVED** [review]

**Comment by 27136773** [comment]
[Missing in HaruNeko](https://github.com/search?q=repo%3Amanga-download%2Fharuneko%20%27agitoon%27&type=code)

**Comment by MikeZeDev** [comment]
Checked yesterday while checking for missing "A" connectors, Agitoon is DEAD apparently, and no alternate URL was found.

**Pull request merged**